ui/imodel-components-react/src/imodel-components-react/timeline/SolarTimeline.scss

```diff
@@ -4,356 +4,136 @@
+  grid-template-columns: auto 1fr auto;
+  grid-template-areas: "solar-timeline-start solar-timeline solar-timeline-end";
 
-    > .loop-playback {
-      color: var(--iui-color-icon-accent);
-      font-size: 16px;
-      cursor: pointer;
-
-      &:focus {
-        outline: none;
-      }
-    }
-
-    > .no-loop-playback {
-      color: var(--iui-color-icon);
-      font-size: 16px;
-      cursor: pointer;
-
-      &:hover {
-        color: var(--iui-color-icon-hover);
-      }
-
-      &:focus {
-        outline: none;
-      }
-    }
+  @media only screen and (max-width: 768px) {
+    grid-template-areas: "solar-timeline solar-timeline" "solar-timeline-start solar-timeline-end";
   }
 
-  .solar-slider {
-    position: relative;
-    display: flex;
-    box-sizing: border-box;
-    align-items: center;
-    flex: 1;
-
-    .rail-outer {
-      position: absolute;
-      top: 50%;
-      transform: translateY(-50%);
-      width: 100%;
-      height: 20px;
-      cursor: pointer;
-    }
-
-    .rail-inner,
-    .rail-solar {
-      position: absolute;
-      top: 50%;
-      transform: translateY(-50%);
-      height: 4px;
-      pointer-events: none;
-    }
-
-    .rail-inner {
-      width: 100%;
-      background: var(--iui-color-background-backdrop);
-    }
-
-    .rail-solar {
-      background: var(--iui-color-background-accent);
-      z-index: 1;
-    }
-
-    .rail-tooltip {
-      position: absolute;
-      margin-left: -11px;
-      top: 0;
-      transform: translateY(-50%);
-      z-index: 1;
-    }
-
-    .solar-handle {
-      position: absolute;
-      top: 50%;
-      z-index: 4;
-      width: 38px;
-      height: 20px;
-      border-radius: var(--iui-border-radius-1);
-      border: 1px solid $buic-background-5;
-      background-image: linear-gradient(
-          to bottom,
-          $buic-background-3,
-          $buic-background-3
-        ),
-        linear-gradient(to bottom, $buic-background-1, $buic-background-1);
-      display: flex;
-      align-items: center;
-      justify-content: center;
-      box-sizing: border-box;
-      transform: translate(-50%, -50%);
-      cursor: pointer;
-
-      > div {
-        width: 1px;
-        height: 10px;
-        opacity: 0.25;
-        margin: 0 1px;
-        background-color: var(--iui-color-text);
-      }
-    }
-
-    .tick-mark {
-      position: absolute;
-      bottom: calc(#{$bottom-position} + 15px);
-      width: 1px;
-      height: 5px;
-      background: rgb(200, 200, 200);
-    }
-
-    .tick-label {
-      position: absolute;
-      bottom: $bottom-position;
-      font-size: 10px;
-      text-align: center;
-    }
-
-    .sunrise,
-    .sunset {
-      font-size: 22px;
-      cursor: default;
-    }
-
-    .sunset {
-      margin: 0px 16px 2px 24px;
-    }
-
-    .sunrise {
-      color: var(--iui-color-icon-warning);
-      margin: 0 24px 0 8px;
+  @media only screen and (max-width: 490px) {
+    .solar-timeline-date-time {
+      display: none;
```

Comment:
I think this is an accessibility concern because the button then has no label.

Reply:
should be changed to visually hidden instead?

Reply:
Copied [the `visually-hidden` CSS](https://github.com/iTwin/iTwinUI/blob/4ff538035cc668efc839576973d2d5e9dd17b141/packages/itwinui-css/src/mixins.scss#L147-L154) from iTwinUI.  There is an [iTwinUI `<VisuallyHidden>` utility component](https://itwinui.bentley.com/docs/visuallyhidden), but am unsure how to make it apply based off a container query being hit or not.